recipe/meta.yaml

```diff
@@ -96,19 +96,30 @@ test:
     {% endfor %}
 
     # Test ncurses library arguments.
-    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib -lncursesw -ltinfow$"
-    - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib -ltinfow$"
-    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib -lncurses -ltinfo$"
-    - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib -ltinfo$"
+    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib .* -lncursesw$"
```

Comment:
The `-ltinfo` check was added for bioconda folks. So we probably should be still checking that somehow.